<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>750</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>905</width>
    <height>748</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Images on the desk</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="12" column="4">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>color3:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="3">
     <widget class="QPushButton" name="addRowButton">
      <property name="text">
       <string>Add row</string>
      </property>
     </widget>
    </item>
    <item row="10" column="4">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>color1:</string>
      </property>
     </widget>
    </item>
    <item row="13" column="5">
     <widget class="QLabel" name="color4">
      <property name="text">
       <string>no color</string>
      </property>
     </widget>
    </item>
    <item row="11" column="4">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>color2:</string>
      </property>
     </widget>
    </item>
    <item row="10" column="5">
     <widget class="QLabel" name="color1">
      <property name="text">
       <string>no color</string>
      </property>
     </widget>
    </item>
    <item row="16" column="0" alignment="Qt::AlignRight">
     <widget class="QLabel" name="FilePath">
      <property name="text">
       <string>File</string>
      </property>
     </widget>
    </item>
    <item row="7" column="4" colspan="3">
     <widget class="QPushButton" name="deliteRowButton">
      <property name="text">
       <string>Delete row</string>
      </property>
     </widget>
    </item>
    <item row="15" column="5">
     <widget class="QLabel" name="imagse_total">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Images total</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4" colspan="3">
     <widget class="QPushButton" name="saveButton">
      <property name="text">
       <string>Save File</string>
      </property>
     </widget>
    </item>
    <item row="15" column="4">
     <widget class="QLabel" name="imagesTotal">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>__</string>
      </property>
     </widget>
    </item>
    <item row="16" column="1">
     <widget class="QLabel" name="pathLable">
      <property name="maximumSize">
       <size>
        <width>16777</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="acceptDrops">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>//</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>Enter color</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="14" column="4">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>color5:</string>
      </property>
     </widget>
    </item>
    <item row="11" column="5">
     <widget class="QLabel" name="color2">
      <property name="text">
       <string>no color</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="nameFilterlineEdit">
      <property name="placeholderText">
       <string>Enter filename</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="13" column="4">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>color4:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="3">
     <widget class="QPushButton" name="loadButto">
      <property name="text">
       <string>Load File</string>
      </property>
     </widget>
    </item>
    <item row="14" column="5">
     <widget class="QLabel" name="color5">
      <property name="text">
       <string>no color</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" rowspan="11" colspan="4">
     <widget class="QTableView" name="tableDetailsView"/>
    </item>
    <item row="12" column="5">
     <widget class="QLabel" name="color3">
      <property name="text">
       <string>no color</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="3">
     <widget class="QGroupBox" name="fareGroupBox">
      <property name="title">
       <string>Filtering by percent</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="fareToLabel">
         <property name="text">
          <string>To</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
        <widget class="QLineEdit" name="linePersFrom">
         <property name="text">
          <string>0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
        <widget class="QLineEdit" name="linePersTo">
         <property name="text">
          <string>100</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSlider" name="SliderPersTo">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="sliderPosition">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="fareFromLabel">
         <property name="text">
          <string>From</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>100</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSlider" name="SliderPersFrom">
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="9" column="4">
     <widget class="QCheckBox" name="checkBoxColorCells">
      <property name="text">
       <string>ShowColors</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="tristate">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Images on the desk</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuAbout"/>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
